gatsby-ssr.js

```diff
@@ -1,26 +1,42 @@
-import React from 'react'
+import React from 'react';
```

Comment:
I don't think there are any substantive changes to this file in this PR. It is just prettier formatting. I'm tempted to revert this file to make it clearer what this PR is and is not actually changing.